docs/accounting/shell.nix

```diff
@@ -0,0 +1,30 @@
+with (import <nixpkgs> {});
```

Comment:
You probably don't want this line given you're pinning the packages below!

Reply:
:man_facepalming: Thanks!